app/src/test/java/com/example/myapplication/CredentialsManagerTest.kt

```diff
@@ -0,0 +1,105 @@
+import com.example.myapplication.CredentialsManager
+import org.junit.Assert.*
+import org.junit.Test
+
+class CredentialsManagerTest {
+
+    private val credentialsManager = CredentialsManager()
+
+    @Test
+    fun testValidEmail() {
+        assertTrue(credentialsManager.isValidEmail("example@test.com"))
+        assertFalse(credentialsManager.isValidEmail("invalid-email"))
+    }
+
+    @Test
+    fun testValidPassword() {
+        assertTrue(credentialsManager.isValidPassword("password123"))
+        assertFalse(credentialsManager.isValidPassword("short"))
+    }
+
+    @Test
+    fun testValidateCredentials() {
+        // Valid email, password, and checkbox checked
+        assertTrue(credentialsManager.validateCredentials("example@test.com", "password123", true))
+
+        // Invalid email
+        assertFalse(credentialsManager.validateCredentials("invalid-email", "password123", true))
+
+        // Invalid password
+        assertFalse(credentialsManager.validateCredentials("example@test.com", "short", true))
+
+        // Checkbox not checked
+        assertFalse(
+            credentialsManager.validateCredentials(
+                "example@test.com",
+                "password123",
+                false
+            )
+        )
+    }
+
+    @Test
+    fun testValidFullName() {
+        assertTrue(credentialsManager.isValidFullName("John Doe"))
+        assertFalse(credentialsManager.isValidFullName(""))
+    }
+
+    @Test
+    fun testValidPhoneNumber() {
+        assertTrue(credentialsManager.isValidPhoneNumber("1234567890"))
+        assertFalse(credentialsManager.isValidPhoneNumber("12345"))
+        assertFalse(credentialsManager.isValidPhoneNumber(""))
+    }
+
+    @Test
+    fun testTermsAccepted() {
+        assertTrue(credentialsManager.isTermsAccepted(true))
```

Comment:
Making a method to test if true is true is a bit too much, is there really a need for it?